scripts/templates/APKBUILD.template

```diff
@@ -16,25 +16,25 @@ subpackages="$pkgname-dev $pkgname-doc"
 source="msquic-$pkgver.tar.gz::ARCHIVE_URI_PLACEHOLDER
 	clog-$_clog.tar.gz::https://github.com/microsoft/CLOG/archive/$_clog.tar.gz
 	gtest-$_gtest.tar.gz::https://github.com/google/googletest/archive/$_gtest.tar.gz
-	openssl3-$_openssl3.tar.gz::https://github.com/quictls/openssl/archive/$_openssl3.tar.gz
+	quictls3-$_quictls3.tar.gz::https://github.com/quictls/openssl/archive/$_quictls3.tar.gz
 	"
 builddir="$srcdir/msquic-SHA_PLACEHOLDER"
 
 prepare() {
 	default_prepare
 
 	cd "$builddir/submodules"
-	rm -rf clog googletest openssl openssl3 xdp-for-windows
+	rm -rf clog googletest openssl quictls3 xdp-for-windows
 	mv ../../CLOG-*/ clog/
 	mv ../../googletest-*/ googletest/
-	mv ../../openssl-*/ openssl3/
+	mv ../../openssl-*/ quictls3/
```

Comment:
Won't the source directory be called "quictls3-*" as well?

Reply:
this actually needs to stay put, because ../../openssl-* is the extracted directory of the archive fetched from this line:
```
quictls3-$_quictls3.tar.gz::https://github.com/quictls/openssl/archive/$_quictls3.tar.gz
```

In the APKBUILD template.  quictls never really broke from using the openssl name in several places, including the commit level archives that this file downloads (i.e. even though we renamed the variable above, the extracted directory path for quictls3-<commit hash>.tar.gz continues to hold the openssl name in its root.

so thats what we have to target when doing the mv operation.